scripts/superdesk/editor/spellcheck/spellcheck.js

```diff
@@ -90,17 +185,25 @@ function SpellcheckService($q, api, dictionaries) {
                 currentOffset = 0,
                 tree = document.createTreeWalker(node, NodeFilter.SHOW_TEXT);
 
+            // Abbreviations found in dictionary.
+            var re = /\w*(?:\.\w*)+/g;
+            var abbrevList = _.words(Object.keys(d), re);
```

Comment:
this should only change on dict updates, so there is no need to run this for every check